### ISSUE TITLE
'Frame rate' is inconsistently spelled as one word

### ISSUE THREAD

**Issue by haykam821** [asiekierka/MinecraftPOMF#4000]
Both the 'frame rate' and 'framerate' spellings are used. Note that the options screen uses the text 'framerate', though that may not be the correct spelling.

**Comment by Octol1ttle** [comment]
> Both the 'frame rate' and 'framerate' spellings are used. Note that the options screen uses the text 'framerate', though that may not be the correct spelling.

The [Wiktionary](https://en.wiktionary.org/wiki/frame_rate) says that "frame rate" is the correct spelling, so which one do we use?

**Comment by modmuss50** [comment]
Going by https://www.minecraft.net/en-us/article/save-energy-with-the-frames-per-second-limiter-feature "framerate" is used, seems like a reasonable source to follow to me.

**Comment by Octol1ttle** [comment]
Maybe it's worth just switching all usages of `framerate` to `fps`, since latter seems to be used much more frequently?
![image](https://github.com/user-attachments/assets/628e1cf5-d185-413f-8a74-8f475de88356)

**Comment by modmuss50** [comment]
No, I dont think FPS and framerate are the same thing. FPS Is a unit of framerate. There might well be places where FPS could be used, but check before blindly replacing it everywhere.

**Comment by Octol1ttle** [comment]
> No, I dont think FPS and framerate are the same thing. FPS Is a unit of framerate. There might well be places where FPS could be used, but check before blindly replacing it everywhere.

they are the same thing in the context of Minecraft, as Minecraft always uses FPS to measure framerate. both instances of "framerate" being used in Yarn refer to FPS (`MAX_FRAMERATE` is used in the max framerate slider, which is measured in FPS, and `frameRateSamples` gets populated with `MinecraftClient#currentFps`)
